src/Components/benchmarkapps/Wasm.Performance/ConsoleHost/Scenarios/GridScenario.cs

```diff
@@ -20,8 +20,8 @@ public GridScenario() : base("grid")
     protected override async Task ExecuteAsync(ConsoleHostRenderer renderer, int numCycles)
     {
         var gridType = _gridTypeOption.HasValue()
-            ? (GridRendering.RenderMode)Enum.Parse(typeof(GridRendering.RenderMode), _gridTypeOption.Value(), true)
-            : GridRendering.RenderMode.FastGrid;
+            ? (GridRendering.GridRenderMode)Enum.Parse(typeof(GridRendering.GridRenderMode), _gridTypeOption.Value(), true)
+            : GridRendering.GridRenderMode.FastGrid;
```

Comment:
I suppose this is a breaking change in a super niche sense, but we have to accept that otherwise we couldn't extend the base class at all. If there's a lot of pushback in previews we can consider options.

Reply:
Since we decided to go with `AssignedRenderMode` instead of just `RenderMode` for the property, hopefully it will be less breaking.